### DISCUSSION TITLE
Intégration iframe dépôt d'offre alternance

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2761] [Idées]
La bonne alternance est une startup d'État de Pôle Emploi qui propose notamment une plateforme de dépôt de besoin pour le recrutement d'alternants : https://labonnealternance.pole-emploi.fr/acces-recruteur. Cette demande est ensuite diffusée sur 1jeune1solution, ParcoursSup, et communiquée aux CFA.

Ils proposent déjà une intégration iframe, par exemple : https://www.akto.fr/deposer-une-offre-demploi-en-alternance/

Nous pouvons intégrer cette iframe sur le simulateur d'embauche sur le modèle de ce que l'on fait avec place des entreprises #1286. Il faudrait afficher le bloc uniquement lorsque la réponse « apprentissage » a été donnée.

La première question du parcours est le SIRET de l'entreprise que l'on pourrait pré-remplir s'il a déjà été renseigné dans notre simulateur.

cc @Waltenberg03

**Comment by johangirod** [comment]
Il faudrait que l'intégration marche aussi pour les iframes. Je pense qu'il faudra revoir la manière d'ajouter ces informations : la solution « Bloc » prend beaucoup d'espace cognitif et éditorial, et n'est pas adaptée aux iframe. L'ajout de CTA en dessous du simulateur atteint peut-être également ses limites : ça serait le quatrième sans hiérarchie.

Quelques pistes de réflexions :
- Passer par une notification du simulateur lors de la réponse apprentissage ?
- Passer par un bandeau léger ?
- Ajouter dans les ressources utiles ?

**Comment by Waltenberg03** [comment]
Un bandeau léger ou un lien en back link simple vers La bonne alternance serait suffisant je pense.

**Comment by johangirod** [comment]
A voir avec #2731
